### PR TITLE
Add Starburst Enterprise on Google Cloud

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: nativeRun.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+printf "\n\nStarting local Jekyll build and serving site \n\n"
+
+bundle install
+bundle exec jekyll clean
+bundle exec jekyll serve --trace --incremental $1
```
Comment (jhlodin) [inline nativeRun.sh:8]: This doesn't seem related to the PR, should probably revert changes to this file

**Review by mosabua — CHANGES_REQUESTED** [review]
Please remove the nativeRun.sh script from the PR. Also confirm text with @k-haley1

File: resources.md [hunk 2]
```diff
@@ -344,11 +344,15 @@ Azure Blob Storage.
...
 <div markdown="1" class="item">
 
-#### [Trino on Qubole on GCP](https://console.cloud.google.com/marketplace/details/qubole-public/qubole-data-service)
+#### [Starburst Enterprise on Google Cloud](https://www.starburst.io/platform/deployment-options/starburst-on-google-cloud/)
+
+Starburst Enterprise is available via the Google Cloud Marketplace.
+
+#### [Trino on Qubole on Google Cloud](https://console.cloud.google.com/marketplace/details/qubole-public/qubole-data-service)
 
 Qubole offers a managed Trino service on GCP that is optimized
```
[the remark this answers is not in the document]
Comment (Jessie212) [reply resources.md:357]: @k-haley1 Changed product name to Google Cloud to match Google's docs.

File: resources.md [hunk 3]
```diff
@@ -344,11 +344,15 @@ Azure Blob Storage.
 
 </div>
 
-### GCP {#gcp}
+### Google Cloud Platform {#gcp}
```
Comment (mosabua) [inline resources.md:347]: Use just Google Cloud here ..

**Review by mosabua — APPROVED** [review]

File: resources.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ show_hero: true
 * [Enterprise support](#support)
 * [Managed services](#managed)
 * [Cloud](#cloud)
-  * [AWS](#aws)
-  * [Azure](#azure)
-  * [GCP](#gcp)
+  * [Amazon Web Services](#aws)
+  * [Microsoft Azure](#azure)
+  * [Google Cloud](#gcp)
```
Comment (Ordinant) [inline resources.md:27]: The list used to be alphabetical so that we didn't betray favoritism. To preserve that with the new wording, lines 25 and 26 need to swap places.

**Review by Ordinant — COMMENTED** [review]
Minor suggestion.

File: resources.md (hunk 4 above)
Comment (mosabua) [reply resources.md:27]: Yes.. lets fix that @Jessie212

Comment (Jessie212) [reply resources.md:26]: Fixed.

**Comment by mosabua** [comment]
Please update the PR title and the commit message @Jessie212 

Add Starburst Enterprise on Google Cloud

**Review by mosabua — APPROVED** [review]

**Comment by mosabua** [comment]
Could you merge this @martint ?

File: resources.md [hunk 5]
```diff
@@ -14,16 +14,15 @@ show_hero: true
...
 * [Management tools](#tools)
 * [Enterprise support](#support)
 * [Managed services](#managed)
 * [Cloud](#cloud)
-  * [AWS](#aws)
-  * [Azure](#azure)
-  * [GCP](#gcp)
+  * [Amazon Web Services](#aws)
+  * [Google Cloud](#gcp)
+  * [Microsoft Azure](#azure)
```
Comment (martint) [inline resources.md:25]: This should be in a separate commit. It's unrelated to adding the Starburst entry

File: resources.md [hunk 6]
```diff
@@ -344,11 +343,15 @@ Azure Blob Storage.
 
 </div>
 
-### GCP {#gcp}
+### Google Cloud {#gcp}
```
Comment (mosabua) [inline resources.md:346]: take that change out as well .. and then send another PR to rejig and fast follow .. or do a separate commit in this PR

**Review by electrum — APPROVED** [review]

File: resources.md [hunk 7]
```diff
@@ -348,6 +348,9 @@ Azure Blob Storage.
 
 <div markdown="1" class="item">
 
+#### [Starburst Enterprise on Google Cloud](https://www.starburst.io/platform/deployment-options/starburst-on-google-cloud/)
+
+Starburst Enterprise is available via the Google Cloud Marketplace.
 #### [Trino on Qubole on GCP](https://console.cloud.google.com/marketplace/details/qubole-public/qubole-data-service)
```
Comment (electrum) [inline resources.md:354]: Add a blank line before the next section

**Review by electrum — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
